src/commands/push.ts

```diff
@@ -0,0 +1,105 @@
+import {
```

Comment:
Given all the imports added to this new file, are there now any unused imports in `commands.ts`? I can't tell just by looking at this PR, but feels like there is a good chance that at least a few of these imports aren't needed by the remaining code.

Reply:
There are many unused imports in `commands.ts` (unrelated).

The only one that was moved over from `commands.ts` to `push.ts` that is now unused in `commands.ts` is `PROJECT_MANIFEST_FILENAME` from `utils`. I can remove it.